docs/man/git-lfs-config.5.ronn

```diff
@@ -266,6 +266,12 @@ be scoped inside the configuration for a remote.
   https://git-scm.com/docs/git-config#git-config-httplturlgt. To set this value
   per-host: `git config --global lfs.https://github.com/.locksverify [true|false]`.
 
+* `lfs.<url>.contenttype`
```

Comment:
Would it be worth mentioning what the `Content-Type` will default to if this is false?

Reply:
Definitely, that's a great idea! Here's an attempt at doing just that: 1818c594 (docs/man: indicate the default Content-Type header, 2018-07-31)
.